static/js/checks.js

```diff
@@ -64,16 +65,17 @@ $(function () {
         start: [20],
         connect: "lower",
         range: {
-            'min': [60, 60],
-            '33%': [3600, 3600],
-            '66%': [86400, 86400],
-            '83%': [604800, 604800],
-            'max': 2592000,
+            'min': 60,
+            '20%': 1800,
+            '40%': 43200,
+            '60%': 604800,
+            '80%': (2592000 * 6),
+            'max': (2592000 * 12),
         },
         pips: {
             mode: 'values',
-            values: [60, 1800, 3600, 43200, 86400, 604800, 2592000],
-            density: 4,
+            values: [60, 3600, 86400, 2592000, (2592000 * 6), (2592000 * 12)],
```

Comment:
convert (2592000 * 6) and (2592000 * 12) into constants, this will enable changing of the value to be done in one place